src/Framework/ModuleAssets.php

```diff
@@ -53,7 +53,7 @@ protected function buildAssetPath(FilePath $assetPath): ?FilePath
             static::$manifestCache[$manifestKey] = json_decode(file_get_contents($manifestPath), true);
         }
         $manifest = static::$manifestCache[$manifestKey];
-        $assetKey = $assetPath->relativeTo($this->productionBuildPath);
+        $assetKey = $assetPath->relativeTo($this->productionBuildPath->value());
```

Comment:
I don't know if this change is necessary. `FilePath` has a `__toString()` method that automatically calls `value()` if it's being cast as a string.